test/system/150-login.bats

```diff
@@ -91,6 +91,37 @@ function setup() {
     assert "$output" =~ "Error: options for paths to the credential file and to the Docker-compatible credential file can not be set simultaneously"
 }
 
+@test "podman login - check with --config global option" {
+    dockerconfig=${PODMAN_LOGIN_WORKDIR}/docker
+    rm -rf $dockerconfig
+
+    registry=localhost:${PODMAN_LOGIN_REGISTRY_PORT}
+
+    run_podman --config $dockerconfig login \
+        --username ${PODMAN_LOGIN_USER} \
+        --password ${PODMAN_LOGIN_PASS} \
+        $registry
```

Comment:
Suggestion:
```diff
     run_podman ... login \
+        --tls-verify=false \
         --username ...
```

Reply:
@edsantiago thanks! Have made a new MR adding that part in